gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/FeatureSupportTest.java

```diff
@@ -587,7 +587,7 @@ public void shouldSupportRemoveEdgesIfEdgeCanBeRemoved() throws Exception {
         public void shouldSupportRemovePropertyIfAPropertyCanBeRemoved() throws Exception {
             try {
                 final Vertex v = graph.addVertex();
-                final Edge e = v.addEdge("self", v);
+                final Edge e = v.addEdge("self", v, "name", "foo");
```

Comment:
good bug fix - i guess we haven't run afoul of that because most graphs will support property removal on edge.